project/AkkaBuild.scala

```diff
@@ -136,7 +136,7 @@ object AkkaBuild extends Build {
     id = "akka-cluster-sharding",
     base = file("akka-cluster-sharding"),
     dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm",
-        persistence % "compile;test->provided", distributedData % "compile;test->provided", clusterTools)
+        persistence % "compile;test->provided", distributedData % "provided;test", clusterTools)
```

Comment:
(Can you add a comment here describing what happens in the pom? Maybe some sbt wizard could help here at some point)